src/olympia/activity/admin.py

```diff
@@ -2,30 +2,35 @@
 
 from olympia.amo.admin import AMOModelAdmin
 from olympia.reviewers.models import ReviewActionReason
+from olympia.zadmin.admin import related_single_content_link
 
 from .models import ActivityLog, ReviewActionReasonLog
 
 
 class ActivityLogAdmin(AMOModelAdmin):
     list_display = (
         'created',
-        'user',
-        '__str__',
+        'user_link',
+        'pretty_arguments',
+        'ip_address',
     )
     raw_id_fields = ('user',)
     readonly_fields = (
         'created',
         'user',
-        '__str__',
+        'pretty_arguments',
+        'ip_address',
     )
     date_hierarchy = 'created'
     fields = (
         'user',
         'created',
-        '__str__',
+        'pretty_arguments',
+        'ip_address',
```

Comment:
TODO: while we're at it, also display if it's an activity we are going to keep forever or not.